src/ChargeTrapping/ChargeTrapping.jl

```diff
@@ -1,6 +1,7 @@
 abstract type AbstractChargeTrappingModel{T <: SSDFloat} end
 
 function _calculate_signal( 
```

Comment:
For backwards compatibility, I would vote for adding a method that makes sure the old method still works, something like:

```julia
_calculate_signal(ctm::AbstractChargeTrappingModel{T},  args...) where {T <: SSDFloat} = _calculate_signal(ctm, NoChargeTrappingModel{T}, args...)
```
(not 100% sure whether this works or whether this might end up in a `StackOverflowError` and one might need to explicitly define what `args...` is)